website/content/partials/deprecation/internal-counters-tokens-api.mdx

```diff
@@ -0,0 +1,8 @@
+## Internal counters tokens API ((#internal-counters-tokens-api))
+
+| Announced | Expected end of support | Expected removal |
+| :-------: | :---------------------: | :--------------: |
+| N/A       | MAY 2025                | N/A
+
+The `/sys/internal/counters/tokens` endpoint, which returns the number of tokens 
+in a Vault cluster, is deprecated and will not longer be supported after May 2025.
```

Comment:
```suggestion
Vault no longer supports the `/sys/internal/counters/tokens` endpoint, which
returned the number of tokens in a Vault cluster.
```
Style correction: write in active voice, we don't need to specify the EOL date in the text since it's already in the table